include/cudaq/Optimizer/CodeGen/IQMGateSetMapping.td

```diff
@@ -31,12 +31,12 @@ def IsNotNull : Constraint<CPred<"$0 ? true : false">>;
 // phased_r(θ, 0, ...)
 
 def CreateRX : NativeCodeCall<
-    "[&]() -> mlir::Value {"
+    "[&]() -> std::size_t {"
```

Comment:
Any reason these and the above `create*` functions should not just return `void`? Seems that we just return 0 on all of them. Just wondering.

Reply:
Yes. Tablegen generates code to check the results of the ops automatically. It keys off the outs specification, which was changed to a variadic result. In that case, it assumes incorrectly that a variadic result is 1 or more values, when it can and is 0 or more values. By returning a `size_t 0` we are forcing the tablegen generated code to explicitly instantiate an empty range of result values, which is what we need here since the op returned 0 values and the new op returns 0 values.